packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -38,7 +38,7 @@ interface DexForwaderBridgeData {
     }>;
 }
 
-const dexForwarderBridgeDataEncoder = AbiEncoder.create([
+export const dexForwarderBridgeDataEncoder = AbiEncoder.create([
```

Comment:
So we don't need to export this either.

Reply:
Yeah it's true, we seem to mostly disable the option in the tests